waterlog/src/reducers/index.js

```diff
@@ -3,10 +3,15 @@ import sensorReducer from './sensorReducer';
 import segmentsReducer from './segmentsReducer';
 import segmentEventReducer from './segmentEventsReducer';
 import segmentLeaksReducer from './segmentLeaksReducer';
+import leakCostsReducer from './leakCostsReducer';
+import leakLitresReducer from './leakLitresReducer';
 
 export default combineReducers({
     sensors: sensorReducer,
     segments: segmentsReducer,
     events: segmentEventReducer,
-    leaks: segmentLeaksReducer
+    leaks: segmentLeaksReducer,
+    costs: leakCostsReducer,
+    litres: leakLitresReducer
+
```

Comment:
Random empty line can be removed.

Reply:
removed random empty line can be removed